RecoTracker/IterativeTracking/python/DisplacedGeneralStep_cff.py

```diff
@@ -0,0 +1,243 @@
+displacedGeneralStepSeeds = _seedCreatorFromRegionConsecutiveHitsTripletOnlyEDProducer.clone(
+    seedingHitSets = "displacedGeneralStepHitTriplets",
+    SeedComparitorPSet = dict(
+        ComponentName = 'CombinedSeedComparitor',
+        mode = cms.string("and"),
+        comparitors = cms.VPSet(
+            cms.PSet(# FIXME: is this defined in any cfi that could be imported instead of copy-paste?
+                ComponentName = cms.string('PixelClusterShapeSeedComparitor'),
+                FilterAtHelixStage = cms.bool(True),
+                FilterPixelHits = cms.bool(False),
+                FilterStripHits = cms.bool(True),
+                ClusterShapeHitFilterName = cms.string('displacedGeneralStepClusterShapeHitFilter'),
+                ClusterShapeCacheSrc = cms.InputTag("siPixelClusterShapeCache") # not really needed here since FilterPixelHits=False
+            ), 
+            _StripSubClusterShapeSeedFilter.clone()
+        )
+    )
+)
+
+
+
+#----------------------------------------- QUALITY CUTS DURING TRACK BUILDING
+import TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff as _TrajectoryFilter_cff
+_displacedGeneralStepTrajectoryFilterBase = _TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
+    maxLostHits = 1,
+    minimumNumberOfHits = 4,
+    minPt = 1,
+)
+
+displacedGeneralStepTrajectoryFilter = _displacedGeneralStepTrajectoryFilterBase.clone(
+    seedPairPenalty = 1,
+)
+
+
+displacedGeneralStepTrajectoryFilterInOut = displacedGeneralStepTrajectoryFilter.clone(
+    minimumNumberOfHits = 4,
+)
+
+import RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimator_cfi
+displacedGeneralStepChi2Est = RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimator_cfi.Chi2ChargeMeasurementEstimator.clone(
+    ComponentName = cms.string('displacedGeneralStepChi2Est'),
+    nSigma = cms.double(3.0),
+    MaxChi2 = cms.double(10.0),
+    clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight')),
+)
+
+
+
+#----------------------------------------- TRACK BUILDING
+import RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi
+displacedGeneralStepTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
+    MeasurementTrackerName = '',
+    trajectoryFilter = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryFilter')),
+    inOutTrajectoryFilter = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryFilterInOut')),
+    useSameTrajFilter = False,
+    minNrOfHitsForRebuild = 4,
+    maxCand = 2,
+    estimator = cms.string('displacedGeneralStepChi2Est'),
+    maxDPhiForLooperReconstruction = cms.double(2.0),
+    # 0.63 GeV is the maximum pT for a charged particle to loop within the 1.1m radius
+    # of the outermost Tracker barrel layer (with B=3.8T)
+    maxPtForLooperReconstruction = cms.double(0.7) 
```

Comment:
```suggestion
```
do we really need to go after loopers in this iteration? (also, considering that up above there is a cut of 1 GeV).
(no value here means the looper section of the code is not running)